src/console/content.ts

```diff
@@ -393,7 +395,8 @@ class ConsoleContent extends Widget {
           return;
         }
         prompt.model.source = code + isComplete.content.indent;
-        prompt.editor.setCursorPosition(prompt.model.source.length);
+        const editor = prompt.editor.editor;
+        editor.setPosition(editor.getModel().getPositionAt(prompt.model.source.length));
```

Comment:
Again, probably makes more sense to be verbose with the name `cursor position` throughout.